libvast/src/segment_store.cpp

```diff
@@ -103,24 +103,24 @@ segment_store::get(const ids& xs) {
     return error;
   // Process candidates in reverse order for maximum LRU cache hits.
   std::vector<const_table_slice_handle> result;
-  VAST_DEBUG("processing", candidates.size(), "candidates");
+  VAST_DEBUG(this, "processing", candidates.size(), "candidates");
   for (auto cand = candidates.rbegin(); cand != candidates.rend(); ++cand) {
     auto& id = *cand;
     caf::expected<std::vector<const_table_slice_handle>> slices{caf::no_error};
     if (id == builder_.id()) {
-      VAST_DEBUG("looking into builder");
+      VAST_DEBUG(this, "looking into builder");
       slices = builder_.lookup(xs);
     } else {
       segment_ptr seg_ptr = nullptr;
       auto i = cache_.find(id);
       if (i != cache_.end()) {
-        VAST_DEBUG("got cache hit for segment", id);
+        VAST_DEBUG(this, "got cache hit for segment", id);
         seg_ptr = i->second;
       } else {
-        VAST_DEBUG("got cache miss for segment", id);
+        VAST_DEBUG(this, "got cache miss for segment", id);
         auto fname = segment_path() / to_string(id);
         if (auto err = load(sys_, fname, seg_ptr)) {
-          VAST_ERROR("unable to load segment:", sys_.render(err));
+          VAST_ERROR(this, "unable to load segment:", sys_.render(err));
```

Comment:
`failed to load...`

Reply:
What about this one, @tobim?

(By the way, github has now a "resolved" feature where you can check off each addressed review comment.)